docs/getting-started/how-to/use-git-effectively/index.md

```diff
@@ -0,0 +1,21 @@
+This page links to resources that describe how to use Git effectively.
+
+## Using Git with `git`
+
+The `git` command-line tool has a large number of commands.
+However, only a small number are important and commonly used.
+These are summarised in GitHub's "[Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf)".
```

Comment:
thought: This resource is OK, but hasn't been updated since 2014, so don't know if it's actively maintained.

Reply:
Good point, but the commands themselves haven't changed in years.